packages/check-es-compat/bin/cli.mjs

```diff
@@ -46,3 +47,41 @@ async function execute(files) {
 
   return { hasErrors: results.some((result) => result.errorCount > 0) };
 }
+
+function parseArguments(args) {
```

Comment:
Will this handle polyfills such as `{Array,String,TypedArray}.prototype.at` ? (see `rule.js` for more examples)

Reply:
Yes, but as you saw further down this works because `{Array,String,TypedArray}.prototype.at` is hard-coded.